projects/element-theme/src/styles/bootstrap/_buttons.scss

```
@@ -242,3 +242,49 @@ button {
     --btn-border-color: #{semantic-tokens.$element-ui-1};
   }
 }
+
+// Split Button
+.btn-split {
+  display: inline-flex;
+
+  > .btn:first-child {
```

```suggestion
   .btn:first-child {
```

Other locations as well.
Direct selectors prevent consumer from wrapping stuff in a component.
Not very likely here, but I don't see a reason why we would need `>`